vspreview/toolbars/scening/dialog.py

```diff
@@ -121,8 +120,10 @@ def on_end_frame_changed(self, value: Frame | int) -> None:
 
         frame = Frame(value)
 
-        index = self.tableview.selectionModel().selectedRows()[0]
-
+        try:
```

Comment:
Why is the try catch needed?

Reply:
Same reason you added it in 258680889d4212d7c0f15c65f174904d63b4b294. There will be a selection model even if there is nothing selected.

Ideally we just disable these controls until there's a selection, but this is a quick fix in the meantime. I chose to match your try/catch pattern for consistency, but it could easily be replaced with a walrus check for falsey selected rows.